tests/vm_migration_test.go

```diff
@@ -45,7 +45,7 @@ var _ = Describe("VmMigration", func() {
 
 	var sourceVM *v1.VirtualMachine
 
-	var TIMEOUT float64 = 10.0
```

Comment:
Are these timeout increases related to this patch series?  I'm just curious if something about these changes introduce more latency into our functional tests.  It doesn't make sense to me how that would be possible.

Reply:
No, it's independent of these changes. It's reuiqred on my host to make the tests pass at all.